LICENSE

```diff
@@ -0,0 +1,26 @@
+Copyright 2016-2022 PureScript
```

Comment:
The range does not include 2023 because there were have been no changes so far in 2023. I could bump it to 2023 though if you want, I don't think it's too important (I'm not a lawyer though, so I can't say for sure).